index.html

```diff
@@ -254,5 +254,55 @@ <h2 class="partnership-title">Brought to you by DeFiPie in partnership with</h2>
     </div>
   </div>
 </section>
+<section class="join">
+  <div class="max-width join__inner">
+    <div class="join-header">
+      <h2 class="h2"><span class="highlight">Join </span> <br> <span class="join-header__title">Vision Capital <br>
+          and get funded</span>
+      </h2>
+      <h3 class="join-header__subtitle">We'll get back to you within 24 hours</h3>
+    </div>
+    <div class="join-footer">
+      <form action="#">
+        <div class="join-form">
+          <div class="join-form__row">
+            <div class="form-field">
+              <label class="form-field__label">How is your project called?</label>
+              <input class="form-field__input" type="text">
+            </div>
+            <div class="form-field">
+              <label class="form-field__label">Your Email</label>
+              <input class="form-field__input" type="email">
+            </div>
+          </div>
+          <div class="join-form__row">
+            <div class="form-field">
+              <label class="form-field__label">Please tell us a few words about the project</label>
+              <input class="form-field__input" type="text">
+            </div>
+            <div class="form-field">
+              <label class="form-field__label">Telegram name</label>
+              <input class="form-field__input" type="text">
+            </div>
+          </div>
+          <button class="button join-form__button">Send</button>
+        </div>
+      </form>
+      <div class="join-info">
+        <div class="join-info__logos">
+          <img alt="facebook" src="images/join/Facebook.png">
+          <img alt="twitter" src="images/join/twitter.png">
+          <img alt="telegram" src="images/join/telegram.png">
+          <img alt="github" src="images/join/github.png">
+        </div>
+        <div class="qqq">
+          <div class="join-info__qqq__email">hello@visioncapital.com
```

Comment:
there is a better approach to handle deep nesting: https://scalablecss.com/bem-nesting-grandchild-elements/#1391467

In our case I would use something like:
```css
.join__email {}
.join__telegram {}
```